app/src/main/java/org/mozilla/fenix/components/metrics/Metrics.kt

```diff
@@ -56,9 +58,15 @@ sealed class Event {
             get() = mapOf("autocomplete" to autoCompleted.toString())
     }
 
-    data class PerformedSearch(val fromSearchSuggestion: Boolean) : Event() {
+    data class PerformedSearch(val fromSearchSuggestion: Boolean, val fromSearchShortcut: Boolean) : Event() {
         override val extras: Map<String, String>?
-            get() = mapOf("search_suggestion" to fromSearchSuggestion.toString())
+            get() = mapOf("search_suggestion" to fromSearchSuggestion.toString(),
```

Comment:
Probably a comment here abt not collecting custom user-added search engines.

Reply:
Both good points. I've added comments in both of these places to ensure we don't track that 😄